tests/foreman/cli/test_docker.py

```diff
@@ -1495,7 +1484,7 @@ def test_positive_upload_image(self, module_org, default_sat):
             compute_resource = make_compute_resource({
                 'organization-ids': [module_org.id],
                 'provider': 'Docker',
-                'url': f'http://{docker_host.ip_addr}:2375',
+                'url': f'http://{docker_contenthost.ip_addr}:2375',
```

Comment:
should this be container_contenthost?

Reply:
Oh, how could I have missed that? Good catch!

Reply:
Probably because it was inside a comment and refactor does not edit comments...
Fixed.